Eatery/Onboarding/Controllers/OnboardingViewController.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  OnboardingViewController.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+protocol OnboardingViewControllerDelegate {
+    func onboardingViewControllerDidTapNextButton(viewController: OnboardingViewController)
+}
+
+class OnboardingViewController: UIViewController {
+
+    internal var stackView: UIStackView!
+
+    internal var titleLabel: UILabel!
+    internal var subtitleLabel: UILabel!
+    internal var imageView: UIImageView!
+    internal var nextButton: UIButton!
+
+    var delegate: OnboardingViewControllerDelegate?
+
+    let model: OnboardingModel!
+
+    init(model: OnboardingModel,
+         nibName nibNameOrNil: String?,
+         bundle nibBundleOrNil: Bundle?) {
+      self.model = model
```

Comment:
Google [style guide](https://google.github.io/swift/#function-declarations) suggests

```
init(
  model: OnboardingModel,
  nibName nibNameOrNil: String?,
  bundle nibBundleOrNil: Bundle?
) {
  // ...
}
```

Actually, is there even a case where the nib name or bundle are not nil?

Reply:
lets use do:
```
init(model: OnboardingModel) {
  self.model = model
  super.init(nibName: nil, bundle: nil)
}
```